src/avfoundation.cs

```diff
@@ -9039,6 +9039,28 @@ interface AVCaptureMetadataOutputObjectsDelegate {
 	}
 #endif
 
+	[NoTV, NoMac, NoWatch, iOS (12,0)]
+	[Internal]
+	[Static]
+	interface AVCapturePhotoSettingsThumbnailFormatKeys {
+		[Field ("AVVideoCodecKey")]
+		NSString CodecKey { get; }
+
+		[Field ("AVVideoWidthKey")]
+		NSString WidthKey { get; }
+
+		[Field ("AVVideoHeightKey")]
+		NSString HeightKey { get; }
+	}
+
+	[NoTV, NoMac, NoWatch, iOS (12,0)]
+	[StrongDictionary ("AVCapturePhotoSettingsThumbnailFormatKeys")]
+	interface AVCapturePhotoSettingsThumbnailFormat {
+		NSString Codec { get; set; }
```

Comment:
are those map to constants ?

Reply:
If so I do not think we have generator support for smart enums in strong dictionaries  but he might be able to use a Wrap property instead.

Reply:
Hmmmm let me see if I can find the values.

Reply:
Wrapping won't work. The generator will expect a key for the added property. But I can deal with it via a partial class (I think).